CMakeLists.txt

```diff
@@ -20,4 +20,7 @@ endif()
 
 include_directories(api/include)
 add_subdirectory(api)
+include_directories(sdk/include)
```

Comment:
Can/should this be pushed down to src/CMakeLists.txt?

Reply:
I could see wanting to have those include directories available to the top-level examples.

Is there a way to do that if we push the include_directories into a sub-CMakeLists.txt?

Reply:
I don't know. I haven't used CMake much, sorry.

Reply:
I guess leave this as-is for now.